frontend/src/app/_components/page/header/header.component.html

```diff
@@ -14,7 +14,7 @@
 
   <ul class="header-navigation-bar">
     <li class="header-navigation" *ngFor="let item of items">
-      <a routerLink="app/documentation/navigate/{{item.hierarchy}}/"
+      <a routerLink="app/documentation/navigate/{{item.hierarchy}}/" (click)="navigateTo(item)"
```

Comment:
You should remove the routerLink if routing is handled by the `navigateTo` method

Reply:
si on l'enleve la class css active n'est plus activée pour la route active.... ([ngClass]="rla.isActive ?)